server/tests/backends/fixtures/dissolve/pandas_advanced.json

```diff
@@ -0,0 +1,265 @@
+{
+    "exclude": [
+        "mysql",
+        "postgres",
+        "snowflake",
+        "mongo"
+    ],
+    "step": {
+        "pipeline": [
+            {
+                "name": "dissolve",
+                "groups": [
+                    "country"
+                ],
+                "aggregations": [
+                    {
+                        "columns": ["inhabitants"],
+                        "agg_function": "avg",
+                        "new_columns": ["inhabitants_avg"]
+                    },
+                    {
+                        "columns": ["region"],
+                        "agg_function": "count distinct",
+                        "new_columns": ["region_distinct_count"]
+                    }
+                ],
+                "include_nulls": true
+            }
+        ]
+    },
+    "input": {
+        "schema": "geojson",
```

Comment:
The table schema standard does provide a way to declare a geometry column:
https://specs.frictionlessdata.io/table-schema/#geojson
Should we use this instead?